scout/server/blueprints/variants/controllers.py

```diff
@@ -946,6 +946,10 @@ def variant_verification(store, mail, institute_obj, case_obj, user_obj, variant
                         transcript_line.append(urllib.parse.unquote(tx_obj['protein_sequence_name']))
                     else:
                         transcript_line.append('')
+                    if refseq_id in gene_obj['common']['primary_transcripts']:
```

Comment:
I think it's fine but I remember we had some bug before because not all gene_obj had the 'common' key. I think it won't hurt here if we check this first, something like:
```
if  gene_obj.get('common') and if refseq_id in gene_obj['common']['primary_transcripts']:
```

Reply:
what about:

```python
if refs_id in gene_obj.get('common',{}).get('primary_transcripts',{}):
    ...
```

Reply:
I've updated the branch according to moonsos suggestion.